inlong-manager/manager-client-tools/src/test/resources/create_group.json

```diff
@@ -0,0 +1,98 @@
+{
+  "groupInfo": {
+    "groupName": "test_group14",
+    "inlongGroupId": "b_test_group45",
+    "description": "",
+    "inCharges": "admin",
+    "proxyClusterId": "2",
+    "mqType": "PULSAR",
+    "mqBaseConf": {
```

Comment:
That MQ config was changed, please change it with the latest branch, thanks.

Reply:
resolve